cli/azd/CHANGELOG.md

```diff
@@ -98,7 +108,7 @@ We plan to improve this behavior with [[#1126]](https://github.com/Azure/azure-d
 
 - [[#979]](https://github.com/Azure/azure-dev/pull/979) Fix provisioning template with non string outputs.
 
-## 0.3.0-beta.4 (2022-10-25) **DEPRECATED**
+## 0.3.0-beta.4 (2022-10-25)
```

Comment:
@danieljurek 👀 the tooling has decided this needs to be a live version

Reply:
Let me have a look at the script generating that and see if we can have it ignore that heading.